flink-quickstart/flink-quickstart-scala/src/main/resources/archetype-resources/pom.xml

```diff
@@ -171,6 +171,11 @@ under the License.
 						</goals>
 					</execution>
 				</executions>
+				<configuration>
+					<args>
+						<arg>-nobootcp</arg>
```

Comment:
I guess it is not so easy to only activate this flag if Java 11 is used, right?

Reply:
well we could add a profile that adds this flag, but it would be inconsistent with the main Flink pom. And given that this is required since Java 9 I think it's fine to have it always on.

Reply:
Actually, yes it would be tricky to make it java version dependent and future-proof.

Ideally, we'd have it in by default (future-proof) and remove it for java 8, but that's kinda tricky. Removing things based on a profile only works in some cases.

Reply:
Not really, what I do know is that we're using this argument for all Flink builds since 1.7 and haven't had a problem with it.

Reply:
Ok, thanks for the info. I assume this is ok to use then.